src/pure.js

```diff
@@ -70,21 +70,9 @@ const render = (
     container,
     component,
     debug: (el = container) => console.log(prettyDOM(el)),
-    rerender: (options) => {
-      if (componentCache.has(component)) component.$destroy()
-
-      // eslint-disable-next-line no-new
-      component = new ComponentConstructor({
-        target,
-        ...checkProps(options)
-      })
-
-      containerCache.add({ container, target, component })
-      componentCache.add(component)
-
-      component.$$.on_destroy.push(() => {
-        componentCache.delete(component)
-      })
+    rerender: async (options) => {
```

Comment:
I don't think we need to accept all options here, I think we should just accept props.

This would match the signature of vue-testing-library's [rerender](https://testing-library.com/docs/vue-testing-library/api#rerenderprops) method, so I think the sibling consistency would be worthwhile

Reply:
I agree only passing `props` makes sense. However, this is a change that will break the code of anybody using `rerender`, so I'm adding a small check that will see if the user passed `{ props: { ... } }` and warn them that's deprecated. And we can hard phase out that signature on the next major update.